packages/server/assets/workspacesCore/typedefs/workspaces.graphql

```diff
@@ -46,13 +46,12 @@ type WorkspaceMutations {
     @hasScope(scope: "workspace:create")
   delete(workspaceId: String!): Boolean! @hasScope(scope: "workspace:delete")
   update(input: WorkspaceUpdateInput!): Workspace! @hasScope(scope: "workspace:update")
-  """
-  TODO: `@hasWorkspaceRole(role: WORKSPACE_ADMIN)` for role changes
-  """
-  updateRole(input: WorkspaceRoleUpdateInput!): Workspace!
+  updateRole(input: WorkspaceRoleUpdateInput!): Boolean!
```

Comment:
@cdriesler 

1. This should return Workspace not Boolean, so that the client (frontend) can immediately ask back for the updated Workspace.team without having to refetch https://www.apollographql.com/docs/react/data/mutations/#include-modified-objects-in-mutation-responses
2. Why not have the same resolver for update/delete like we do for projects?

Reply:
1. Done, makes good sense to me
2. I did not catch that this was the pattern and like it much better, done